test/e2e/framework/util.go

```diff
@@ -474,7 +475,7 @@ func ProxyMode(f *Framework) (string, error) {
 			Containers: []v1.Container{
 				{
 					Name:    "detector",
-					Image:   imageutils.GetE2EImage(imageutils.Agnhost),
+					Image:   AgnHostImage,
 					Command: []string{"/bin/sleep", "3600"},
```

Comment:
`agnhost` image seems to have a `pause` command for this purpose, instead using sleep https://github.com/kubernetes/kubernetes/tree/master/test/images/agnhost#pause